src/lib/Scenes/City/Components/Event/index.tsx

```diff
@@ -94,8 +102,31 @@ export class Event extends React.Component<Props, State> {
     }
   }
 
+  @track((__, _, args) => {
+    const actionName = args[0]
+    const slug = args[1]
+    const id = args[2]
+    return {
+      action_name: actionName,
+      action_type: Schema.ActionTypes.Tap,
+      owner_type: Schema.OwnerEntityTypes.Show,
+      owner_id: id,
+      owner_slug: slug,
+    } as any
```

Comment:
this too

Reply:
Yeah i looked at this with @l2succes and it was acting weird with anything but `any`, interested in revisiting it - we're using it all over the analytics

Reply:
Yeah this isn’t good. Can you file a ticket that this should be cleaned up post launch and include the TS error it throws?